Neos.Neos/Resources/Private/Translations/en/Modules.xlf

```diff
@@ -2,15 +2,6 @@
 <xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
 	<file original="" product-name="Neos.Neos" source-language="en" datatype="plaintext">
 		<body>
-			<trans-unit id="clickToEdit" xml:space="preserve">
-				<source>Click to edit</source>
-			</trans-unit>
-			<trans-unit id="clickToDelete" xml:space="preserve">
-				<source>Click to delete</source>
-			</trans-unit>
-			<trans-unit id="cancel" xml:space="preserve">
-				<source>Cancel</source>
-			</trans-unit>
```

Comment:
you did it right, was just saying that some other module might use it from the `Modules.xlf` file, so you need to search for occurrences of `clickToEdit`, `clickToDelete` and `cancel` and make sure they use `Main.xlf` and not `Modules.xlf`